packages/mobile/locales/base/translation.json

```diff
@@ -666,6 +666,17 @@
     "step2": "With {{appName}}, it’s easy to access and easy to grow your crypto.",
     "step3": "All you need is a mobile phone to send, spend and explore."
   },
+  "enableBiometry": {
+    "title": "Secure your wallet",
+    "description": "Face ID secures your wallet and makes it easier to use Valora",
+    "cta": {
+      "useFaceId": "Turn on Face ID",
+      "useTouchId": "Turn on Touch ID",
+      "useFingerprint": "Turn on fingerprint",
+      "useFace": "Turn on face unlock",
+      "useIris": "Turn on iris recognition"
```

Comment:
i've been searching for what the common phrase for the android fingerprint/face/iris but couldn't find :(